yaml/wp_base.yaml

```diff
@@ -317,6 +325,10 @@ packages:
   HYSTERESE_WW:                   !include { file: wp_number.yaml, vars: { property: "HYSTERESE_WW"               , min: "2.0", max: "10.0", step:   "0.1", unit: "°C"}}
   NE_STUFE_WW:                    !include { file: wp_number.yaml, vars: { property: "NE_STUFE_WW"                , min: "0", max: "3" }}
 
+  COP_WW:                         !include { file: wp_energy_cop.yaml, vars: { property: "COP_WW"    , aufnahmeleistung_summe_kwh: "EL_AUFNAHMELEISTUNG_WW_SUMME_KWH"   , waermeertrag_tag_kwh: "WAERMEERTRAG_WW_TAG_KWH"   , waermeertrag_tag_wh: "WAERMEERTRAG_WW_TAG_WH"   }}
+  COP_HEIZ:                       !include { file: wp_energy_cop.yaml, vars: { property: "COP_HEIZ"  , aufnahmeleistung_summe_kwh: "EL_AUFNAHMELEISTUNG_HEIZ_SUMME_KWH" , waermeertrag_tag_kwh: "WAERMEERTRAG_HEIZ_TAG_KWH" , waermeertrag_tag_wh: "WAERMEERTRAG_HEIZ_TAG_WH" }}
```

Comment:
What do you think about combining WAERMEERTRAG_HEIZ_TAG_KWH and WAERMEERTRAG_HEIZ_TAG_WH and publishing this value to HA as kWh? So that we just keep these 2 as internal sensors. I did the same for the EL_AUFNAHMELEISTUNG_XXX_SUMME_KWH. We don't need to keep the format of the heatpump and everyone would need to combine it again. It is a leftover from the very beginning. I can also do it ... just tell me how to continue ;)

Reply:
That would certainly simplify things and make more sense in my opinion. On one hand, I didn’t want to mess too much with your code, and on the other hand, I’m not very proficient with lambda. It would certainly make sense for the COP calculation to be only triggered when one of the values changes. I just didn’t know how to implement that and the following code did not work. 
`on_value: 
    then:  
      - component.update: COD_WW` 
 
 Therefore, I opted for the static update based on time.